sod-server/src/rooms/schema/Player.ts

```diff
@@ -23,7 +23,30 @@ export class Player extends Schema {
     return this.roads.filter((road) => !road.edge)
   }
 
-  cards(state: GameState) {
+
+  payToBank(state: GameState, cardType: string, payAmount: number){
+    const cards = this.cardsOfType(state, cardType)
+    const numberOfCards = cards.length
+
+    if(numberOfCards < payAmount){
+      return
+    }
+
+    for (let index = 0; index < payAmount - 1; index++) {
+      cards [index].owner = null;
+    }
+  }
+
+  cards(state: GameState){
+    return state.deck.filter((card) => card.owner === this.id)
+  }
+
+  cardsOfType(state: GameState, cardType: string){
+    const cards = this.cards(state)
+    return cards.filter((card) => card.type === cardType)
+  }
+
+  numberOfCards(state: GameState) {
```

Comment:
nú tú hevur gjørt cards rætt, og cardsOfType, so kan hetta slettast. Ístaðin kann man brúka cardsOfType().length